node/node.go

```diff
@@ -513,6 +527,11 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, chainDBFactory shardc
 		Interface("genesis block header", node.Blockchain().GetHeaderByNumber(0)).
 		Msg("Genesis block hash")
 
+	node.incomingMessages = make(chan incomingMessage, RxQueueSize)
+	for i := 0; i < RxWorkers; i++ {
```

Comment:
why explicit for loop, not range

Reply:
There is no channel/array/slice to range over; Go does not support plain integer range like in Python either.